src/libsystemd/sd-json/json-util.c

```diff
@@ -306,6 +306,26 @@ int json_dispatch_pidref(const char *name, sd_json_variant *variant, sd_json_dis
         return 0;
 }
 
+int json_dispatch_ifindex(const char *name, sd_json_variant *variant, sd_json_dispatch_flags_t flags, void *userdata) {
+        int *ifi = ASSERT_PTR(userdata), r, t;
+
+        if (sd_json_variant_is_null(variant)) {
```

Comment:
I think we should accept null only when SD_JSON_RELAX is set.

Reply:
we have SD_JSON_REFUSE_NULL already for that, that systematically refuses "null" for this field